src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mcfooter/MixedContentFooter.java

```diff
@@ -0,0 +1,72 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.mcfooter;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class MixedContentFooter extends WikiBasePageObject{
+
+  @FindBy(css = "#mixed-content-footer")
+  private WebElement mcFooter;
+
+  @FindBy(css = ".mcf-card-wiki-articles")
+  private WebElement moreOfWikiArticlesCard;
+
+  @FindBy(css = ".mcf-card-discussions")
+  private WebElement discussionsCard;
+
+  @FindBy(css = ".mcf-card-related-wikis")
+  private WebElement exploreWikisCard;
+
+  @FindBy(css = ".mcf-card-article__link")
+  private List<WebElement> articleLinks;
+
+  public MixedContentFooter(WebDriver driver) {
```

Comment:
if the only thing that this constructor does is calling super() method then it is not needed. Please remove it